content/en/docs/reference/using-api/cel.md

```diff
@@ -113,6 +113,11 @@ Example CEL expressions:
 CEL is configured with the following options, libraries and language features,
 introduced at the specified Kubernetes versions:
 
+{{< note >}}
+CEL libraries are available for use starting one minor release after they are introduced.
+For example, a library introduced in Kubernetes 1.32 would be available in Kubernetes 1.33.
+{{< /note >}}
```

Comment:
Could we edit the available version above to make it the right version instead of adding this note which might confuse the user? e.g. Semver is available 1.34+; IP is available 1.31+, etc. 

It looks all good after this one is addressed^^